chainer/function_set.py

```diff
@@ -94,7 +102,8 @@ def parameters(self):
         The order of parameters is consistent with :meth:`gradients` property.
 
         """
-        return sum((func.parameters for _, func in self._get_sorted_funcs()), ())
+        return sum(
+            (func.parameters for _, func in self._get_sorted_funcs()), ())
```

Comment:
Sorry. There was this line more than 79 characters(pep8 e501). So I have put a new line.